.travis.yml

```diff
@@ -22,3 +30,7 @@ deploy:
     distributions: "bdist_wheel"
     on:
       tags: true
+  - provider: script
+    script: ssh deploy-api@api.openfisca.fr
```

Comment:
I guess `ssh deploy-api@api.openfisca.fr` is called by `ssh deploy-api@api.openfisca.fr`